src/current/v24.1/molt-fetch.md

```diff
@@ -114,18 +137,29 @@ Complete the following items before using MOLT Fetch:
 | `--metrics-listen-addr`                       | Address of the metrics endpoint.<br><br>**Default:** `'127.0.0.1:3030'`                                                                                                                                                                                                                                                                                                                                                   |
+| `--non-interactive`                           | Run the fetch process without interactive prompts. This is recommended **only** when running `molt fetch` in an automated process (i.e., a job or continuous integration).                                                                                                                                                                                                                                                |
+| `--ongoing-replication`                       | Enable continuous [replication](#replication) to begin after the fetch process succeeds (i.e., initial source data is loaded into CockroachDB).                                                                                                                                                                                                                                                                           |
 | `--pglogical-replication-slot-drop-if-exists` | Drop the replication slot, if specified with `--pglogical-replication-slot-name`. Otherwise, the default replication slot is not dropped.                                                                                                                                                                                                                                                                                 |
-| `--pglogical-replication-slot-name`           | The name of a replication slot to create before taking a snapshot of data (e.g., `'fetch'`). This flag is only necessary if you want to use a replication slot other than the default slot.                                                                                                                                                                                                                               |
+| `--pglogical-replication-slot-name`           | The name of a replication slot to create before taking a snapshot of data (e.g., `'fetch'`). **Required** in order to perform continuous [replication](#replication) from a source PostgreSQL database.                                                                                                                                                                                                                   |
```

Comment:
Do these `pg` ones get ignored if importing from MySQL?

Reply:
@ZhouXing19 what's the behavior here (if you pass the pglogical flags but have MySQL source)?

Reply:
Yes I think so, but might want to confirm with @ryanluu12345 who has more context here.

Reply:
Yes. It should be ignored since we won't enter the code path that uses these.